java/arcs/core/storage/handle/HandleManager.kt

```diff
@@ -136,4 +212,72 @@ class HandleManager(
             dereferencer = RawEntityDereferencer(schema, aff?.dereferenceFactory())
         ).also { storageProxy.registerHandle(it) }
     }
+
+    /**
+     * @deprecated Use [rawEntitySetHandle] instead.
+     */
+    /* ktlint-disable max-line-length */
+    @Deprecated(
+        "Use rawEntitySetHandle instead",
+        replaceWith = ReplaceWith("this.rawEntitySetHandle(storageKey, schema, callbacks, name, refinement, ttl, canRead)")
+    )
+    /* ktlint-enable max-line-length */
+    suspend fun setHandle(
+        storageKey: StorageKey,
+        schema: Schema,
+        callbacks: SetCallbacks<RawEntity>? = null,
+        name: String = storageKey.toKeyString(),
+        refinement: Refinement<RawEntity>? = null,
+        ttl: Ttl = Ttl.Infinite,
+        canRead: Boolean = true
+    ): SetHandle<RawEntity> = rawEntitySetHandle(
+        storageKey,
+        schema,
+        callbacks,
+        name,
+        refinement,
+        ttl,
+        canRead
+    )
+
+    /**
+     * Creates a new [SetHandle] which manages a singleton of type: [Reference], where the
+     * [Reference] is expected to *reference* a [RawEntity] described by the provided [Schema].
+     */
+    // TODO: support refinement here.
```

Comment:
Do we even want to support refinement on a collection of references?

Reply:
I think @Cypher1 might know.